tests/preprocessing/test_util.py

```diff
@@ -88,3 +91,15 @@ def test_index_dtype_with_None(self):
         returned_df = _explode_agg("id", "c", orig_df, agg_df)
         solution_df = pd.DataFrame(orig)
         assert_frame_equal(returned_df, solution_df)
+
+
+class TestAngleCentroidMultipoints:
+    """Test util method angle_centroid_multipoints"""
+
+    a = Point((130, 45))
+    b = MultiPoint([[160, 10], [-170, 20]])
+    c = MultiPoint([[20, 0], [30, 10], [40, 20]])
+    g = gpd.GeoSeries([a, b, c])
+    g_solution = gpd.GeoSeries([a, Point([175, 15]), Point([30, 10])])
```

Comment:
It would be nice if you (also) use an example from a referenced source. E.g., [https://rosettacode.org/wiki/Averages/Mean_angle](https://rosettacode.org/wiki/Averages/Mean_angle).
Or if your example is already from a credible source than to add the reference